libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.GetConversationReference.schema

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+    "$role": "implements(Microsoft.IDialog)",
+    "title": "Get ConversationReference for current activity context.",
+    "description": "Gets the ConversationReference from current context and stores in property so it can be used to with ContinueConversation action.",
```

Comment:
```suggestion
    "description": "Gets the ConversationReference from current context and stores it in a property so it can be used with the ContinueConversation action.",
```